docs/public-networks/how-to/connect/manage-peers.md

```diff
@@ -13,6 +13,23 @@ Hyperledger Besu peer-to-peer (P2P) discovery happens periodically based on the
 
 The frequency of discovery isn't configurable, but you can [limit remote connections](#limit-remote-connections) in public networks and [randomly prioritize connections](../../reference/cli/options.md#random-peer-priority-enabled) in small, stable networks.
 
+## The peering process
+
+The peering process requires the port (default 30303) to be open to UDP and TCP traffic to the world (0.0.0.0/0). The `discovery stack` uses UDP to keep things lightweight and quick and only
```

Comment:
do we want to say something about network security here - you would have firewalls or whatever to set up to limit traffic outside of this